### ISSUE TITLE
Field names on map

### ISSUE THREAD

**Issue by kevinhughes27** [kevinhughes27/tournament-app#27]
it would be nice to put the field name on the field on the map

**Comment by theoandtheb** [comment]
Wouldn't it get crowded on a mobile device?  If we adopt the on-hover & on-touch or on-target handler I suggested before we may be able to make it possible to pair names with fields without overlaying info that is already on the page.

**Comment by kevinhughes27** [comment]
yeah that could work too. How crowded would depend on the name length I think. If its something like `field 1` I think it would be okay

**Comment by mmasse** [comment]
OCUA has a long and short name for fields, the short name would work better
on mobile.
Not sure that will scale to other tournaments though.

Below is the setup for Ben Franklin
There are multiple Ben Franklin Fields
Ben Franklin Park 1 is also BFP1

Ben Franklin ParkNameBen Franklin ParkCodeBFPRegionWest
<http://www.ocua.ca/zuluru/regions/view/region:4>Address180 Knoxdale Rd.
CityOttawa  ProvinceOntario  SurfaceGrassStatusOpenDriving Directions

Fields are located to the North of Hunt Club West, and to the East of
Knoxdale, in Ben Franklin Park. From the Queensway, take the Greenbank exit
South, turn left at West Hunt Club, then left again on Knoxdale. Fields and
parking are on your right.
Private Instructions
Dogs are allowed on leash on the recreational pathways and grass between
sportfields but not on the sportsfields or children's play areas.
Spectators are NOT allowed to set up along the centre of the soccer fields.
Sidelines should be set up on the opposite side.
Absolutely no alcohol
